src/renderer/features/wallets/WalletSelect/ui/WalletGroup.tsx

```diff
@@ -28,6 +30,7 @@ export const WalletGroup = ({ type, wallets }: Props) => {
           <WalletIcon type={type} />
           <CaptionText className="text-text-secondary  font-semibold uppercase">{t(GroupLabels[type])}</CaptionText>
           <CaptionText className="text-text-tertiary font-semibold">{wallets.length}</CaptionText>
+          {type === WalletType.PROXIED && <ProxiedTooltip />}
```

Comment:
I suggest to do this:
- remove `type`
- use the first wallet
This way UI won't bother business logic details like: `type === WalletType.PROXIED`
```javascript
const walletType = wallets[0].type as WalletFamily;
{walletUtils.isProxied(wallet[0]) && <ProxiedTooltip />}
```